examples/pytorch/IterativePruning/README.md

```diff
@@ -0,0 +1,48 @@
+## Iterative Pruning
+Pruning is the process of compressing a neural network that involves removing weights from a trained model.
+It could be achieved by either removing the neurons in a specific layer or make the weights of the connections 
```

Comment:
```suggestion
Example pruning techniques include removing the neurons within a specific layer, or setting the weights of connectons that are already near zero to zero. This script applies the latter technique. Pruning a model reduces its size, at the cost of worsened model accuracy.
```

Proofreading suggestion - also, is there a definition of iterative pruning from the pytorch docs we could link to here?

Reply:
Updated the string

Reply:
Added a link to https://pytorch.org/tutorials/intermediate/pruning_tutorial.html